src/App.jsx

```diff
@@ -16,6 +17,7 @@ const App = () => {
           <Route path="/home" element={<Home />} />
           <Route path="/all-articles" element={<ArticlesList />} />
           <Route path="/articles/:articleId" element={<ArticlePage />} />
+          <Route path="/articles/:articleId/comments" element={<CommentsList />} />
```

Comment:
I can see why you have created this path but would it make more sense to see the comments in the same view as the articles?

Reply:
Actually I can see further down that is what you are doing. If you don't need this path anymore then remove it. If you do envisage using this path then you'll need to get the `artilce_id` from the URL as it is currently expected on props.